src/providers/web3.provider.ts

```diff
@@ -1,53 +1,113 @@
 import Web3 from 'web3';
-import ABI_V2 from '@cli/shared/v2.abi.json';
+import ABI_SSV_NETWORK_CORE from '@cli/shared/abi.ssv-network.json';
+import ABI_SSV_NETWORK_VIEWS from '@cli/shared/abi.ssv-network-views.json';
 
 export default class Web3Provider {
-  static BLOCK_RANGE_500K = 10000;
+  static BLOCK_RANGE_500K = 500_000;
```

Comment:
Did you check that it works fine in all the cases with this block range? Might be that in some cases we need logic of decreasing block range until it stop failing.
This is very important question for such a critical tool as liquidator, so let's make sure we have an answer and follow-up task to improve it if necessary